tests/CredentialRequestClient.spec.ts

```diff
@@ -71,14 +37,20 @@ describe('Credential Request Client ', () => {
       .withFormat('jwt_vc')
       .withCredentialType('https://imsglobal.github.io/openbadges-specification/ob_v3p0.html#OpenBadgeCredential')
       .build();
-    const proof: ProofOfPossession = await createProofOfPossession({
-      issuerURL: credReqClient.getCredentialEndpoint(),
-      jwtSignerArgs: jwtArgs,
-      jwtSignerCallback: (args) => signJWT(args),
-      jwtVerifyCallback: (args) => verifyJWT(args),
-    });
+    const proof: ProofOfPossession = await new ProofOfPossessionBuilder()
+      .withPoPCallbackOpts({
+        proofOfPossessionCallback: proofOfPossessionCallbackFunction,
+        proofOfPossessionCallbackArgs: {
+          kid: 'did:example:123',
+          payload: {
+            aud: 'aud',
+            iss: 'sphereon',
+          },
+        },
+      })
+      .build();
     const credentialRequest: CredentialRequest = await credReqClient.createCredentialRequest(proof);
-    expect(credentialRequest.proof.jwt).toContain(partialJWT);
+    expect(credentialRequest.proof.jwt).toContain('ey.val.eu');
```

Comment:
I'm wondering why the actual call was changed to a mocked one. This is a working example to the user.